src/api.ts

```diff
@@ -13,3 +13,28 @@ export function getPeople(): Promise<Person[]> {
     .then(() => fetch(API_URL))
     .then(response => response.json());
 }
+
+export function addParent(people: Person[]) {
+  function getParent(parentName: string) {
+    return people.find(person => person.name === parentName);
+  }
```

Comment:
This function can be exported as a separate helper instead of being stored in the middle of another function
```suggestion
  function getParent(people, parentName: string) {
    return people.find(({ name }) => name === parentName);
  }
```